src/lib/containers/markdown/MarkdownEditor.tsx

```diff
@@ -55,13 +57,19 @@ export const MarkdownEditor: React.FunctionComponent<MarkdownEditorProps> = ({
       const textBeforeTag = text.substring(0, start)
       const textAfterTag = text.substring(start, text.length)
       setSelectionStart(start + user.length + 1)
-      newText.push(textBeforeTag, `@${user.replace(/\s/g, '')}`, textAfterTag)
+      if (tagModalWithKeyboard) {
+        newText.push(textBeforeTag, `${user.replace(/\s/g, '')}`, textAfterTag)
+      } else {
+        newText.push(textBeforeTag, `@${user.replace(/\s/g, '')}`, textAfterTag)
```

Comment:
Could you embed this condition instead (single line to reduce duplicate code that may drift)?
Something like
newText.push(textBeforeTag, tagModalWithKeyboard ? '' : '@', user.replace(/\s/g, ''), textAfterTag)